Tests/SentryTests/SentryOptionsTest.m

```diff
@@ -401,6 +401,24 @@ - (void)testInvalidAttachStackTrace
     XCTAssertEqual(NO, options.attachStacktrace);
 }
 
+- (void)testStitchAsyncCodeDisabledPerDefault
+{
+    SentryOptions *options = [self getValidOptions:@{}];
+    XCTAssertEqual(NO, options.stitchAsyncCode);
+}
+
+- (void)testStitchAsyncCodeEnabled
+{
+    SentryOptions *options = [self getValidOptions:@{ @"stitchAsyncCode" : @YES }];
+    XCTAssertEqual(YES, options.stitchAsyncCode);
+}
+
+- (void)testInvalidStitchAsyncCode
+{
+    SentryOptions *options = [self getValidOptions:@{ @"attachStacktrace" : @"Invalid" }];
+    XCTAssertEqual(NO, options.attachStacktrace);
+}
```

Comment:
H: you're asserting `attachStacktrace` instead of `stitchAsyncCode`

Reply:
Good catch thanks